### PR TITLE
Added set, remove, get, show, pretty in a 'handle' group

### PR DESCRIPTION
**Pull request by cfalas**
This solves issue #60

### CODE REVIEW

**Comment by krofna** [comment]
@meooow25 should `_updateroles` even be in this group?

**Comment by cfalas** [comment]
I assumed that because it has to do with roles, therefore handles it should be here, but it's not that related so idk

**Comment by cfalas** [comment]
Made the changes mentioned (removed _updateroles from the group and renamed show to list)

File: tle/cogs/handles.py [hunk 1]
```diff
@@ -124,9 +124,13 @@ class Handles(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
-    @commands.command(brief='sethandle [name] [handle] (admin-only)')
+    @commands.group(brief='Commands that have to do with handles', invoke_without_command=True)
+    async def handle(seld, ctx):
```
Comment (meooow25) [inline tle/cogs/handles.py]: Small typo, `self`.
Also, if you add `await ctx.send_help(ctx.command)` to the body, the command group will send its help message when invoked without subcommands.

File: tle/cogs/handles.py [hunk 2]
```diff
@@ -124,9 +124,13 @@ class Handles(commands.Cog):
...
+        """Change or collect information about specific handles on Codeforces"""
+
+    @handle.command(brief='set [name] [handle] (admin-only)')
```
Comment (meooow25) [inline tle/cogs/handles.py]: Would be nice to update all the `brief`s since you're working in this area.

**Pull request merged**